README.rst

```diff
@@ -63,7 +63,8 @@ Base Images
 ~~~~~~~~~~~
 
 The "base" Dockerfile encompass the installation of the framework and all of the dependencies
-needed.
+needed. It is needed before building container for TensorFlow 1.8.0 and before. Container of TensorFlow 1.9.0
```

Comment:
Replace "container" with "images" in both places on this line. Also, say 1.9.0 and above.
So something like:
"Building a base image is not required for images for TensorFlow 1.9.0 and onwards."